templates/nextflow-ecs-task-definition.yaml

```diff
@@ -134,6 +140,12 @@ Resources:
               HostPort: 6379
           Command:
             - --appendonly yes
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-region: !Sub '${AWS::Region}'
+              awslogs-group: !Ref TowerTaskLogGroup
+              awslogs-stream-prefix: 'tower'
```

Comment:
`tower` appears multiple times in this file.  might be nice to make it a parameter?

Reply:
Done!